models/engine/db_storage.py

```diff
@@ -74,3 +74,16 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """Method to retrieve one ojbect"""
```

Comment:
Maybe add a little bit more information to the doc string about what the function returns and why.